iotdb-api/pipe-api/src/main/java/org/apache/iotdb/pipe/api/event/dml/insertion/TsFileInsertionEvent.java

```diff
@@ -34,4 +34,7 @@ public interface TsFileInsertionEvent extends Event, AutoCloseable {
    * @return {@code Iterable<TabletInsertionEvent>} the list of {@link TabletInsertionEvent}
    */
   Iterable<TabletInsertionEvent> toTabletInsertionEvents();
+
+  /** The method is used to prevent circular replication in PipeConsensus */
+  boolean isGeneratedByPipeConsensus();
```

Comment:
This is pipe api and is exposed to users... Do not include this in interface.

Reply:
ok